### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
# Description

Found via `codespell -S .mypy_cache -L ream,selfs,froms,devision,fo` and `typos --format brief`.

### CODE REVIEW

File: ropetest/refactor/__init__.py [hunk 1]
```diff
@@ -908,7 +908,7 @@ def a_func(self, var):
                 self.mod, self.mod.read().index("var") + 1
             )
 
-    # NOTE: This situation happens alot and is normally not an error
+    # NOTE: This situation happens a lot and is normally not an error
     # @testutils.assert_raises(RefactoringError)
     def test_not_rais_exception_when_there_is_a_field_with_the_same_name(self):
```
Comment (lieryan) [inline ropetest/refactor/__init__.py:913]:
```suggestion
    def test_not_raise_exception_when_there_is_a_field_with_the_same_name(self):
```

File: ropetest/objectinfertest.py [hunk 2]
```diff
@@ -398,11 +398,12 @@ class C2(object):
         self.assertEqual(c1_class, a_var.get_type())
         self.assertEqual(c2_class, b_var.get_type())
 
-    def test_we_know_the_type_of_catched_exceptions(self):
+    def test_we_know_the_type_of_caught_exceptions(self):
         code = dedent("""\
             class MyError(Exception):
                 pass
-            try:
+            try::w
```
Comment (lieryan) [inline ropetest/objectinfertest.py:405]:
```suggestion
            try:
```

File: rope/base/worder.py [hunk 3]
```diff
@@ -55,8 +55,8 @@ def get_word_at(self, offset):
     def get_primary_range(self, offset):
         return self._context_call("get_primary_range", offset)
 
-    def get_splitted_primary_before(self, offset):
-        return self._context_call("get_splitted_primary_before", offset)
+    def get_split_primary_before(self, offset):
+        return self._context_call("get_split_primary_before", offset)
```
Comment (lieryan) [inline rope/base/worder.py:59]:
```suggestion
    def get_splitted_primary_before(self, offset):
        return self._context_call("get_splitted_primary_before", offset)
```

This is public interface, so shouldn't be fixed.

File: rope/base/worder.py [hunk 4]
```diff
@@ -250,7 +250,7 @@ def get_primary_at(self, offset):
         start, end = self.get_primary_range(offset)
         return self.raw[start:end].strip()
 
-    def get_splitted_primary_before(self, offset):
+    def get_split_primary_before(self, offset):
```
Comment (lieryan) [inline rope/base/worder.py:253]:
```suggestion
    def get_splitted_primary_before(self, offset):
```

File: rope/contrib/codeassist.py [hunk 5]
```diff
@@ -66,7 +66,7 @@ def starting_offset(source_code, offset):
 
     """
     word_finder = worder.Worder(source_code, True)
-    expression, starting, starting_offset = word_finder.get_splitted_primary_before(
+    expression, starting, starting_offset = word_finder.get_split_primary_before(
```
Comment (lieryan) [inline rope/contrib/codeassist.py:69]:
```suggestion
    expression, starting, starting_offset = word_finder.get_splitted_primary_before(
```

File: rope/contrib/codeassist.py [hunk 6]
```diff
@@ -361,7 +361,7 @@ def sorted_proposals(proposals, scopepref=None, typepref=None):
 def starting_expression(source_code, offset):
     """Return the expression to complete"""
     word_finder = worder.Worder(source_code, True)
-    expression, starting, starting_offset = word_finder.get_splitted_primary_before(
+    expression, starting, starting_offset = word_finder.get_split_primary_before(
```
Comment (lieryan) [inline rope/contrib/codeassist.py:364]:
```suggestion
    expression, starting, starting_offset = word_finder.get_splitted_primary_before(
```

File: rope/contrib/codeassist.py [hunk 7]
```diff
@@ -390,7 +390,7 @@ def __init__(
             self.expression,
             self.starting,
             self.offset,
-        ) = self.word_finder.get_splitted_primary_before(offset)
+        ) = self.word_finder.get_split_primary_before(offset)
```
Comment (lieryan) [inline rope/contrib/codeassist.py:393]:
```suggestion
        ) = self.word_finder.get_splitted_primary_before(offset)
```

File: rope/contrib/fixsyntax.py [hunk 8]
```diff
@@ -62,7 +62,7 @@ def old_pyname():
         new_code = pymodule.source_code
 
         def new_pyname():
-            newoffset = self.commenter.transfered_offset(offset)
+            newoffset = self.commenter.transferred_offset(offset)
```
Comment (lieryan) [inline rope/contrib/fixsyntax.py:65]:
```suggestion
            newoffset = self.commenter._transferred_offset(offset)
```

This probably shouldn't be a public method.

File: rope/contrib/fixsyntax.py [hunk 9]
```diff
@@ -97,7 +97,7 @@ def comment(self, lineno):
             self._set(line, self.lines[start])
         self._fix_incomplete_try_blocks(lineno, indents)
 
-    def transfered_offset(self, offset):
+    def transferred_offset(self, offset):
```
Comment (lieryan) [inline rope/contrib/fixsyntax.py:100]:
```suggestion
    def _transferred_offset(self, offset):
```

File: ropetest/codeanalyzetest.py [hunk 10]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
         code = "A()"
         self.assertEqual("A()", self._find_primary(code, 2))
 
-    def test_splitted_statement(self):
+    def test_split_statement(self):
         word_finder = worder.Worder("an_object.an_attr")
         self.assertEqual(
-            ("an_object", "an_at", 10), word_finder.get_splitted_primary_before(15)
+            ("an_object", "an_at", 10), word_finder.get_split_primary_before(15)
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:225]:
```suggestion
            ("an_object", "an_at", 10), word_finder.get_splitted_primary_before(15)
```

File: ropetest/codeanalyzetest.py [hunk 11]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
...
         )
 
-    def test_empty_splitted_statement(self):
+    def test_empty_split_statement(self):
         word_finder = worder.Worder("an_attr")
-        self.assertEqual(("", "an_at", 0), word_finder.get_splitted_primary_before(5))
+        self.assertEqual(("", "an_at", 0), word_finder.get_split_primary_before(5))
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:230]:
```suggestion
        self.assertEqual(("", "an_at", 0), word_finder.get_splitted_primary_before(5))
```

File: ropetest/codeanalyzetest.py [hunk 12]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
...
 
-    def test_empty_splitted_statement2(self):
+    def test_empty_split_statement2(self):
         word_finder = worder.Worder("an_object.")
         self.assertEqual(
-            ("an_object", "", 10), word_finder.get_splitted_primary_before(10)
+            ("an_object", "", 10), word_finder.get_split_primary_before(10)
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:235]:
```suggestion
            ("an_object", "", 10), word_finder.get_splitted_primary_before(10)
```

File: ropetest/codeanalyzetest.py [hunk 13]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
...
         )
 
-    def test_empty_splitted_statement3(self):
+    def test_empty_split_statement3(self):
         word_finder = worder.Worder("")
-        self.assertEqual(("", "", 0), word_finder.get_splitted_primary_before(0))
+        self.assertEqual(("", "", 0), word_finder.get_split_primary_before(0))
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:240]:
```suggestion
        self.assertEqual(("", "", 0), word_finder.get_splitted_primary_before(0))
```

File: ropetest/codeanalyzetest.py [hunk 14]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
...
 
-    def test_empty_splitted_statement4(self):
+    def test_empty_split_statement4(self):
         word_finder = worder.Worder("a_var = ")
-        self.assertEqual(("", "", 8), word_finder.get_splitted_primary_before(8))
+        self.assertEqual(("", "", 8), word_finder.get_split_primary_before(8))
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:244]:
```suggestion
        self.assertEqual(("", "", 8), word_finder.get_splitted_primary_before(8))
```

File: ropetest/codeanalyzetest.py [hunk 15]
```diff
@@ -219,33 +219,33 @@ def test_functions_on_ending_parens(self):
...
 
-    def test_empty_splitted_statement5(self):
+    def test_empty_split_statement5(self):
         word_finder = worder.Worder("a.")
-        self.assertEqual(("a", "", 2), word_finder.get_splitted_primary_before(2))
+        self.assertEqual(("a", "", 2), word_finder.get_split_primary_before(2))
```
Comment (lieryan) [inline ropetest/codeanalyzetest.py:248]:
```suggestion
        self.assertEqual(("a", "", 2), word_finder.get_splitted_primary_before(2))
```

**Comment by lieryan** [comment]
Thank you for the contribution, @kianmeng

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**